CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 
 ## Older releases:
   * [CHANGELOG-1.8.md][18]
+  * [CHANGELOG-1.8.md][17]
```

Comment:
Looks like 1.7 changelog item missed, and 1.9 changelog should also be added.

Reply:
thanks!

1.9 changelog is in Line 2
1.7 changelog  item has been corrected